marketplace_builder/pages/getting-started/users/users.liquid

```diff
@@ -0,0 +1,37 @@
+---
+converter: markdown
+metadata:
+  title: Users
+slug: get-started/users/users
+---
+
+A **User** is an account that any of your users can have. Each user is assumed to have a unique email address. 
+ 
+**User Profiles** are roles in the marketplace. Each User Profile can be associated with any number of Custom Attributes, Categories, and Custom Model Types. 
+
+## The default User Profile
+
+All users are assigned a User Profile named Default. If you don't want to distinguish one type of user from the other, you don't need to create any other user profiles. 
+
+## Sign up forms
+
+**Sign up forms** allow users to sign up by entering their information in a form embedded in a page. 
+
+All forms should be located in the `marketplace_builder/form_configurations` directory.   
+
+{% include alert/note.html content="Check out our <a href="">API and Form Configurations</a> pages to learn more about sign up forms." %}
```

Comment:
@diana-lakatos Please use new syntax:

`+{% include "alert/note", content: "Check out our <a href="">API and Form Configurations</a> pages to learn more about sign up forms." %}`